smithy-templates.json

```diff
@@ -130,6 +130,26 @@
         "gradlew.bat",
         ".gitignore"
       ]
+    },
+    "simple-trait": {
+      "documentation": "Trait package for simple Smithy structure trait with multiple inputs.",
+      "path": "examples/trait-package-plugin/create-simple-trait",
+      "include": [
+        "gradle/",
+        "gradlew",
+        "gradlew.bat",
+        ".gitignore"
+      ]
+    },
+    "existing-trait": {
```

Comment:
Q: do we have a generated trait with custom validation example?

Reply:
We do. In the smithy examples package.